.github/actions/clang-format/entrypoint.sh

```diff
@@ -1,14 +1,19 @@
 #!/bin/sh
 
-set -e
-
 git config --global --add safe.directory /github/workspace
 
-set +e
+# Fix any formatting.
+git-clang-format "${INPUT_TARGET}"
+
+git diff
+
+# Now revert any changes to auto-generated files.
+find . -name *.pb-c.c -exec git checkout --quiet '{}' \;
+find . -name *.pb-c.h -exec git checkout --quiet '{}' \;
+find . -name *.proto -exec git checkout --quiet '{}' \;
```

Comment:
(lint) Quote the parameter to -name so the shell won't interpret it. [SC2061]
(lint) Use ./*glob* or -- *glob* so names with dashes won't become options. [SC2035]